microsoft-edge/webview2/code-samples-links.md

```diff
@@ -0,0 +1,60 @@
+| Create a WebView2 in a Win32 application | [Get started with WebView2 in Win32 apps](./get-started/win32.md) | Starter code at [WebView2 Samples Win32 Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/Win32_GettingStarted) |
+| Set up a WebView2 in a WPF application | [Get started with WebView2 in WPF apps](./get-started/wpf.md) | .NET starter code at [WebView2 Samples WPF Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/WPF_GettingStarted) |
+| Create a WebView2 in a WinForms application | [Get started with WebView2 in WinForms apps](./get-started/winforms.md) | .NET starter code at [WebView2 Samples WinForms Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/WinForms_GettingStarted) |
+| Set up a WebView2 in a WinUI 3 application | [Get started with WebView2 in WinUI 3 (Windows App SDK)](./get-started/winui.md) | Starter code at [WebView2 Samples WinUI Getting Started (github.com)](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/GettingStartedGuides/WinUI3_GettingStarted) |
+| Create a WebView2 in WinUI 2 application | [Get started with WebView2 in WinUI 2 apps guide](./get-started/winui2.md) | Uses the WinUI 2 NuGet package.  Has no sample code. |
+
+
+<!-- ====================================================================== -->
+## WebView2 sample apps
+
+WebView2 sample apps include solutions for Win32, WPF, WinForms, and WinUI.  The Microsoft Visual Studio solution file [WebView2Samples.sln](https://github.com/MicrosoftEdge/WebView2Samples/blob/master/SampleApps/WebView2Samples.sln) gathers the following samples.
+
+When you load `WebView2Samples.sln` into Visual Studio, the following projects appear in Solution Explorer, as folders:
+
+| Type of sample | Sample Project | Notes and source code location |
+|---|---|---|
+| Win32 C++ | `WebView2APISample` <!-- below, section 1 "Win32 C++ samples" --> | Includes the code in the [WebView2APISample](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/SampleApps/WebView2APISample) folder. |
+| Win32 C++ | `WebView2SampleWinComp` <!-- below, section 1 --> | Includes the code in the [WebView2SampleWinComp](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/SampleApps/WebView2SampleWinComp) folder. |
+| Windows Presentation Foundation (WPF) .NET | `WebView2WindowsFormsBrowser` <!-- below, section 2, "Windows Presentation Foundation (WPF) .NET samples" --> | Uses code in the [WebView2WpfBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/SampleApps/WebView2WpfBrowser) folder. |
+| Windows Presentation Foundation (WPF) .NET | `WebView2WpfBrowser` <!-- below, section 2 --> | The .NET Windows Forms Sample is in the [WebView2WindowsFormsBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/SampleApps/WebView2WindowsFormsBrowser) folder. |
```

Comment:
correction: swap the right-hand column's cell contents for row 3 & 4, they are reversed.  I'd follow the order shown in Solution Explorer screen capture.